src/devices/Bmp180/Bmp180.cs

```diff
@@ -0,0 +1,304 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//Ported from https://github.com/adafruit/Adafruit_Python_BMP/blob/master/Adafruit_BMP/BMP085.py  
+//Formulas and code examples can also be found in the datasheet https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf
+
+using System;
+using System.Device.I2c;
+using System.Buffers.Binary;
+using System.Threading;
+
+namespace Iot.Device.Bmp180
+{
+    public class Bmp180:IDisposable
+    {
+        private I2cDevice _i2cDevice;
+        private readonly CommunicationProtocol _communicationProtocol;
+        private bool _initialized = false;
+        private readonly CalibrationData _calibrationData;
+        private Sampling _mode = Sampling.Standard;
+
+        private enum CommunicationProtocol
+        {
+            I2c
+        }
+
+        public Bmp180(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;            
+            _calibrationData = new CalibrationData();
+            _communicationProtocol = CommunicationProtocol.I2c;
+        }
+
+        private void Begin()
+        {
+            _initialized = true;

```

Comment:
NIT: Should we set `_initialized = true` after reading the calibration data? That way if there are two threads trying to read temperature at the same time, you won't run into issues where the calibration data hasn't been read yet.

Reply:
I agree with set the `_initialized = true` after reading the calibration data, but I was take as example  the `Begin()` function of the bmp280. 

https://github.com/dotnet/iot/blob/3ebec0ba37323275347cffbcbaac1ba1feb7859f/src/devices/Bmp280/Bmp280.cs#L41-L54

Reply:
I see, yeah that seems like a bug on Bmp280, we should fix that 😄